hw/ip/prim/dv/prim_lfsr/data/prim_lfsr_cov_excl.el

```diff
@@ -0,0 +1,49 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+//==================================================
+// This file contains the Excluded objects
+// Generated By User: udij
+// Format Version: 2
+// Date: Mon Oct  5 14:44:08 2020
+// ExclMode: default
+//==================================================
+// Coverage exclusion file for prim_lfsr
+CHECKSUM: "1437046094"
+INSTANCE: prim_lfsr_tb.gen_duts[24].i_prim_lfsr
+ANNOTATION: "[EXTERNAL/UNR] external seed and entropy inputs are disabled in DV, these assertions are fully covered in FPV"
```

Comment:
UNR means unreachable. That in itself, makes the justification final. Appending 'EXTERNAL' to it should not be necessary. If it is really UNR, does the design need cleanup?
@msfschaffner

Reply:
Also, the point of standardizing the way exclusions are annotated is to be able to `grep '[EXTERNAL]' foo.el` easily. This might create problems.

Reply:
To the first point: Got it, I updated to just list these two exclusions as `[UNR]`. The intent for this LFSR testbench was just to check that all LFSRs are maximum length, so these inputs are unnecessary (this is all fully tested in FPV).
Maybe @msfschaffner can comment more?

To the second point: got it, as mentioned above I updated to just `[UNR]`.

Reply:
Sorry, in that case, I should have been clear. UNR is reserved for something that is truly unreachable, no matter what you do. In this case, it is unreachable by DV choice (you intentionally don't drive those inputs). This should instead be  tagged as `EXTERNAL` since we are claiming that it is covered in FPV (it would be tagged as LOW_RISK if it were not covered even by FPV). Sorry for the back and forth.

Reply:
hm that's true,  my original intent was to show that it was unreachable from DV choice like you said, but I think you're right that changing to EXTERNAL would make it a bit more accurate.